contents/docs/cdp/geoip-enrichment.md

```diff
@@ -61,6 +61,12 @@ set property `$geoip_disable` to `true` on that event.
 
 ## FAQ
 
+### How can I still get some geo data, but with less precision?
+
+We use the Maxmind GeoIP2 City database, in which latitude and longitude are set to the detected city's geographical center. The data is not more precise than the city name field.
+
+If having city precision is too much, then you can use use the [property filter transformation](/docs/cdp/property-filter). The GeoIP transformation adds event and person properties (`$set` and `$set_once`), so you'll want to filter those out as well. For example, if you are trying to filter `$geoip_latitude`, make sure to include `$set.$geoip_latitude` and `$set_once.$geoip_latitude` to the property filter configuration.
```

Comment:
```suggestion
If having city precision is too much, then you can use use the [property filter transformation](/docs/cdp/property-filter). The GeoIP transformation adds event and person properties (`$set` and `$set_once`), so you'll want to filter those out as well. For example, if you are trying to filter `$geoip_latitude`, make sure to include `$geoip_latitude` for the event property and`$set.$geoip_latitude` & `$set_once.$geoip_latitude` for the person properties to the property filter configuration.
```

Reply:
@ivanagas it's important they add all 3 otherwise they will still keep that info in PostHog. not sure on wording

Reply:
good point